.github/workflows/main.yml

```diff
@@ -1521,6 +1521,48 @@ jobs:
           paths: "./*/*/build/test/*/results/TEST-*.xml"
 
 
+  java-debugger-test:
+    name: Java Debugger ${{ matrix.config }} on Linux/JDK ${{ matrix.java }}
```

Comment:
here too,` ${{ matrix.config }}` can be removed since the job doesn't actually have the "config" axis, it only has `java` `[11, 17]`.